RevenueCatUI/CustomerCenter/Views/SubscriptionDetailView.swift

```diff
@@ -34,6 +34,9 @@ struct SubscriptionDetailView: View {
     @Environment(\.navigationOptions)
     var navigationOptions
 
+    @Environment(\.openURL)
+    var openURL
```

Comment:
I think this will never happen but could this cause issues in app extensions? I believe it will never try to open the url in that case since canOpenURL could add return false, but we did add an openURL if not app extension function and I wanted to ask, just in case.

https://github.com/RevenueCat/purchases-ios/pull/4531/files